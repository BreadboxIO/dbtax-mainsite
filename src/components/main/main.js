import { App } from 'components/app/app';
import { ContactBar } from 'components/contact-bar/contact-bar';
import { Navigation } from 'components/navigation/navigation';
import PropTypes from 'prop-types';
import React from 'react';

export const Main = props => {
  const { children } = props;

  return (
    <App>
      <ContactBar />
      <Navigation />
      {children}
    </App>
  );
};

Main.propTypes = {
  children: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node])
};

Main.defaultProps = {
  children: null
};
