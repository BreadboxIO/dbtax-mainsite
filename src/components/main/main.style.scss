@import '../../styles/config/variables';

.root {
  background: $color-primary-lighter;
  -webkit-overflow-scrolling: touch;
  box-sizing: border-box;
  height: 100%;
  overflow-y: auto;
  position: relative;
}
