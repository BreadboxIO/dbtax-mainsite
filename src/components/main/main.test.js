import { Main } from './main';
import { StaticRouter } from 'react-router-dom';
/* global React, create */

describe('<Main />', () => {
  it('should render correctly', () => {
    const component = create(
      <StaticRouter>
        <Main />
      </StaticRouter>
    ).toJSON();
    expect(component).toMatchSnapshot();
  });
});
