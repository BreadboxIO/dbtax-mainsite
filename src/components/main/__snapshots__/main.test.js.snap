// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render correctly 1`] = `
<div
  className="root"
>
  <div
    className="root"
  >
    <div
      className="content"
    >
      <a
        href="tel:+14147647730"
        rel="noopener noreferrer"
        target="_blank"
      >
        <i
          className="fas fa-phone-alt"
        />
         
        414-764-7330
      </a>
      <a
        className="address"
        href="https://www.google.com/maps/place/1125+Milwaukee+Ave,+South+Milwaukee,+WI+53172"
        rel="noopener noreferrer"
        target="_blank"
      >
        <i
          className="fas fa-home"
        />
         
        1125 Milwaukee Ave.
         · 
        South Milwaukee
        , 
        WI
         · 
        53172
      </a>
    </div>
  </div>
  <div
    className="root"
  >
    <div
      className="content"
    >
      <a
        className="logo"
        href="/"
        onClick={[Function]}
      />
      <nav
        className="links"
      >
        <a
          aria-current="page"
          className="active"
          href="/"
          onClick={[Function]}
          style={Object {}}
        >
          About
        </a>
        <a
          aria-current="page"
          className="active"
          href="/"
          onClick={[Function]}
          style={Object {}}
        >
          Services
        </a>
        <a
          aria-current="page"
          className="active"
          href="/"
          onClick={[Function]}
          style={Object {}}
        >
          Client Resources
        </a>
        <a
          aria-current="page"
          className="active"
          href="/"
          onClick={[Function]}
          style={Object {}}
        >
          Legal
        </a>
      </nav>
    </div>
  </div>
</div>
`;
